src/theory/tools/wp/bir_wpScript.sml

```diff
@@ -246,7 +246,7 @@ PAT_X_ASSUM ``bir_is_bool_exp_env s.bst_environ ex``
               (HO_MATCH_MP bir_bool_values thm)
             ) >>
 REV_FULL_SIMP_TAC std_ss [] >>
-FULL_SIMP_TAC std_ss [bir_not_val_true, bir_exec_stmtB_def,
+FULL_SIMP_TAC std_ss [bir_neg_val_true, bir_exec_stmtB_def,
```

Comment:
Are you based on `master`? This [should be](https://github.com/kth-step/HolBA/blob/master/src/theory/bir-support/bir_exp_equivScript.sml#L99) `bir_not_equiv`.

Reply:
https://travis-ci.com/kth-step/HolBA/jobs/200293490#L351

Reply:
Yep, this was a typo resulting from copy/pasting the work from an older branch.